activities/tool-management/miro.md

```diff
@@ -0,0 +1,23 @@
+---
+type: resource
+---
+
+# Miro
+
+Miro is an online collaborative whiteboard platform. Miro built a platform for modern work, enabling collocated, distributed, and remote teams to communicate and collaborate across formats, tools, channels, and time zones - without the constraints of physical location, meeting space, and whiteboard.
+
+## Using Miro
+
+Miro [has extensive documention](https://help.miro.com/hc/en-us).
+
+## Miro management
+
+The Operations Coordinator, Chief Executive and Codebase Stewards can add new users to Miro.
```

Comment:
I actually thing Miro users can add users, or am I wrong?

Reply:
updated it - thanks!

Reply:
@demkodo Where have you updated this?